File: Python/old_approach/old_approach.py
def reverse_number(num):
	the_sum = 0
	digit = 0
	
	while num != 0:
		digit = num % 10
		the_sum = the_sum * 10 + digit
		num = int(num / 10)

	return the_sum

num = int(input("Please insert a number: "))
res = reverse_number(num)
print(res)

File: Ruby/old_approach/old_approach.rb
def reverse_number(num)
	sum = 0
	digit = 0

	while (num != 0)
		digit = num % 10
		sum = sum * 10 + digit
		num = num / 10
	end

	sum
end

print "Please insert a number: "
num = gets.to_i
res = reverse_number num
puts res